types/src/v0/impls/auction.rs

```diff
@@ -291,10 +292,14 @@ impl<TYPES: NodeType> AuctionResultsProvider<TYPES> for SolverAuctionResultsProv
         &self,
         view_number: TYPES::Time,
     ) -> anyhow::Result<TYPES::AuctionResult> {
-        let resp = SurfClient::new(self.0.clone())
-            .get::<TYPES::AuctionResult>(&format!("/v0/api/auction_results/{}", *view_number))
-            .send()
-            .await?;
+        let resp = SurfClient::new(
```

Comment:
This works as so? I'll make a PR for sequencer then. Though I'm wondering if we shouldn't have these paths in configuration.

Reply:
PR https://github.com/EspressoSystems/espresso-sequencer/pull/1853

Reply:
Yes, +1 to putting the paths into a config somewhere!

Reply:
I'll do a separate PR